ingestors/elm.go

```diff
@@ -26,6 +26,7 @@ func (ingestor *Elm) Schedule() string {
 }
 
 func (ingestor *Elm) Ingest() []data.PackageVersion {
+	ingestor.LatestRun = time.Now()
```

Comment:
Can you stash the stuff, set latestrun and then return the stashed stuff. It could explode and then we'd capture the wrong time.